### PR TITLE
Update QUEST_LV_0100.tsv

### PR DESCRIPTION
**Pull request by Sunaries**
Minor update.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -3848,31 +3848,31 @@ QUEST_LV_0100_20150317_003847	Talk to the Battle Commander
...
-QUEST_LV_0100_20150317_003862	Offer worship to the statue
-QUEST_LV_0100_20150317_003863	Try offering worship to Goddess Zemyna Statue. The Statue is located at the end of the road on the right of Laimonas.
-QUEST_LV_0100_20150317_003864	You offered worship like Laimonas said. Tell Laimnoas about it.
+QUEST_LV_0100_20150317_003862	Offer worship to the Goddess Statue.
+QUEST_LV_0100_20150317_003863	Offer worship to the Goddess Zemyna Statue. The statue is located at the end of the road on the right of Laimonas.
+QUEST_LV_0100_20150317_003864	You offered worship like Laimonas said. Tell Laimonas about it.
 QUEST_LV_0100_20150317_003865	Listen to the favor of Laimonas.
-QUEST_LV_0100_20150317_003866	Defeat Infrorocktor around the marked area
-QUEST_LV_0100_20150317_003867	Do Laimonas a favor and defeat the Infro Rocktors on the road to Klaipeda.
+QUEST_LV_0100_20150317_003866	Defeat the Infrorocktor near the marked area.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Infrorocktors

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -3848,31 +3848,31 @@ QUEST_LV_0100_20150317_003847	Talk to the Battle Commander
...
+QUEST_LV_0100_20150317_003867	Do Laimonas a favor and defeat the Infrorocktors on the road to Klaipeda.
 QUEST_LV_0100_20150317_003868	Talk to Klaipeda Guard Captain
 QUEST_LV_0100_20150317_003869	You seem to have taken care of Laimonas' favor. Tell the Guard leader that you got rid of the monsters on the road to Klaipeda.
 QUEST_LV_0100_20150317_003870	Defeat Infrorocktor
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Infrorocktor

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -3839,7 +3839,7 @@ QUEST_LV_0100_20150317_003838	Check status of the Scout
 QUEST_LV_0100_20150317_003839	Please check the status of the attack Scout to Kepa.
 QUEST_LV_0100_20150317_003840	Remind the Scout that there is an order to assemble to Klaipeda
 QUEST_LV_0100_20150317_003841	Remind the Scout again that there is an order to assemble to Klaipeda.
-QUEST_LV_0100_20150317_003842	Defeat the Leaf Bugs and collect luggage
+QUEST_LV_0100_20150317_003842	Defeat the Leaf Bugs and collect the luggage.
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: luggages? Because you are collecting multiple luggages right?

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**